text/trace/0170-sampling-probability.md

```diff
@@ -0,0 +1,705 @@
+	+ [Using head trace probability to count all spans](#using-head-trace-probability-to-count-all-spans)
+    + [Head sampling for traces](#head-sampling-for-traces)
+      - [`Parent` Sampler](#parent-sampler)
+      - [`TraceIDRatio` Sampler](#traceidratio-sampler)
+      - [Dapper's "Inflationary" Sampler](#dappers-inflationary-sampler)
+- [Proposed specification text](#proposed-specification-text)
+- [Recommended reading](#recommended-reading)
+- [Acknowledgements](#acknowledgements)
+
+<!-- tocstop -->
+
+Objective: Specify a foundation for sampling techniques in OpenTelemetry.
+
+## Motivation
+
+Probability sampling allows consumers of sampled telemetry data to
+collect a fraction of telemetry events and use them to estimate total
+quantities about the population of events, such as the total rate of
+events with a particular attribute.
+
+These techniques enable reducing the cost of telemetry collection,
+both for producers (i.e., SDKs) and for processors (i.e., Collectors),
+without losing the ability to (at least coarsely) monitor the whole
+system.
+
+Sampling builds on results from probability theory.  Estimates drawn
+from probability samples are *random variables* that are expected to
+equal their true value.  When all outcomes are equally likely, meaning
+all the potential combinations of items used to compute a sample of
+the sampling logic are equally likely, we say the sample is _unbiased_.
+
+Unbiased samples can be used for after-the-fact analysis.  We can
+answer questions such as "what fraction of events had property X?"
+using the fraction of events in the sample that have property X.
+
+This document outlines how producers and consumers of sample telemetry
+data can convey estimates about the total count of telemetry events,
+without conveying information about how the sample was computed, using
+a quantity known as **adjusted count**.  In common language, a
+"one-in-N" sampling scheme emits events with adjusted count equal to
+N.  Adjusted count is the expected value of the number of events in
+the population represented by an individual sample event.
+
+## Examples
+
+These examples use an attribute named `sampler.adjusted_count` to
+convey sampling probability.  Consumers of spans, metrics, and logs
+annotated with adjusted counts are able to calculate accurate
+statistics about the whole population of events, without knowing
+details about the sampling configuration.
+
+The hypothetical `sampler.adjusted_count` attribute is used throughout
+these examples to demonstrate this concept, although the proposal
+below for OpenTelemetry `Span` messages introduces a dedicated field
+with specific interpretation for conveying head sampling probability.
+
+### Span sampling
+
+Example use-cases for probability sampling of spans generally involve
+generating metrics from spans.
+
+#### Sample spans to Counter Metric
+
+For every complete span it receives, the example processor will synthesize
```

Comment:
I think there was a logical leap here for me.  Are we discussing a `SpanProcessor`, or just a conceptual processor of spans that generates metric data?